packages/platform-android/src/commands/runAndroid/tryLaunchAppOnDevice.js

```diff
@@ -8,31 +8,33 @@
  */
 
 import {spawnSync} from 'child_process';
-import {logger} from '@react-native-community/cli-tools';
+import {logger, CLIError} from '@react-native-community/cli-tools';
 
 function tryLaunchAppOnDevice(
-  device: string,
+  device?: string,
   packageNameWithSuffix: string,
   packageName: string,
   adbPath: string,
   mainActivity: string,
 ) {
   try {
     const adbArgs = [
-      '-s',
-      device,
       'shell',
       'am',
       'start',
       '-n',
       `${packageNameWithSuffix}/${packageName}.${mainActivity}`,
     ];
-    logger.info(
-      `Starting the app on ${device} (${adbPath} ${adbArgs.join(' ')})...`,
-    );
+    if (device) {
+      adbArgs.unshift('-s', device);
+      logger.info(`Starting the app on "${device}"...`);
+    } else {
+      logger.info('Starting the app...');
```

Comment:
... on a simulator?

Reply:
for android it would be emulator, not sure if we want to change that?